Mathlib/LinearAlgebra/Alternating/Basic.lean

```diff
@@ -844,7 +844,7 @@ end MultilinearMap
 
 namespace AlternatingMap
 
-/-- Alternatizing a multilinear map that is already alternating results in a scale factor of `n!`,
+/-- If a multilinear map is already alternating, then its alternation equals `n!` times the map,
```

Comment:
I think the original is better, especially because its referring to `alternization` as opposed to the non-existent, `alternation`.

Reply:
For whatever reason I convinced myself that alternization wasn't established terminology, but I see that it is. I reverted this change.